lib/matplotlib/backend_bases.py

```diff
@@ -1006,6 +1006,10 @@ def get_hatch_linewidth(self):
         """Get the hatch linewidth."""
         return self._hatch_linewidth
 
+    def set_hatch_linewidth(self, hatch_linewidth):
+        """Set the hatch linewidth - not supported on all backends."""
```

Comment:
Which ones are not supported?

Reply:
Was facing some issues with Cairo backends but not anymore